Gopkg.toml

```diff
@@ -19,3 +25,16 @@ required = ["k8s.io/code-generator/cmd/client-gen"]
 [[constraint]]
   name = "github.com/stretchr/testify"
   version = "1.3.0"
+
+[[constraint]]
+  name = "github.com/kubernetes-sigs/kube-batch"
+  branch = "master"
```

Comment:
why we use master branch for kube-batch, because for v0.3 and v0.4 release, the generate code is not updated and the UT will fail because of the generate code of kube-batch. I took a look at that and found that there miss a commit in the v0.3 and v0.4 release about update generate code. So we use the master branch.